lib/view.js

```diff
@@ -37,7 +37,10 @@ export function extractViews(entities) {
 
   Object.keys(entities).forEach((entityId) => {
     const entity = entities[entityId];
-    if (entity.attributes.view) {
+    // Entity is a view if it has the 'view' attribute set.
+    // Consider default_view as view by default. 
+    if (entity.attributes.view ||
+        (entity.entity_id === DEFAULT_VIEW_ENTITY_ID && entity.attributes.view !== false)) {
```

Comment:
Hm, actually it either true or undefined, no way for it to be false (it just won't be set in `group.py`)

I removed the extra check. Now default_view is always considered a view.